apps/web/playwright/booking-pages.test.ts

```diff
@@ -31,11 +100,6 @@ test.describe("pro user", () => {
     await page.click('[data-testid="event-type-link"]');
     // Click [data-testid="incrementMonth"]
     await page.click('[data-testid="incrementMonth"]');
-
-    // @TODO: Find a better way to make test wait for full month change render to end
-    // so it can click up on the right day, also when resolve remove other todos
-    // Waiting for full month increment
-    await page.waitForTimeout(400);
```

Comment:
I think this was removed by mistake. This is a temporal fix made by @alannnc so tests aren't randomly failing due to loading times.

Reply:
sry, removed this by accident